android/app/src/main/java/com/reactnativenavigation/params/SlidingOverlayParams.java

```diff
@@ -17,5 +17,6 @@ public static Position fromString(String string) {
 
     public String screenInstanceId;
     public NavigationParams navigationParams;
+    public Integer autoDismissTimerSec;
```

Comment:
Fair enough. So lets leave it as is. We can always add milliseconds support later